### ISSUE TITLE
Implementing a custom field

### ISSUE THREAD

**Issue by stephenafamo** [charmbracelet/huh#108]
**Is your feature request related to a problem? Please describe.**
It is not possible to implement a custom field because `prevField` and `nextField` are private

**Describe the solution you'd like**
Export `prevField` and `nextField`

**Describe alternatives you've considered**
I cannot think of a way to work around this since Group requires these messages to proceed to the next field or page

**Comment by stephenafamo** [comment]
I've been able to simulate this by creating a `Confirm` field and sending an `enter` key press to it

```go
		_, cmd := huh.NewConfirm().
			WithKeyMap(huh.NewDefaultKeyMap()).
			Update(tea.KeyMsg(tea.Key{Type: tea.KeyEnter}))
		return f, cmd
```

Would be nice not to have to do this.

**Comment by ardnew** [comment]
To be clear, it is possible to _implement_ a custom `Field`. It is just not possible for that `Field` to send a message to change focus away from your `Field`.

**Comment by stephenafamo** [comment]
> It is just not possible for that `Field` to send a message to change focus away from your `Field`.

This makes the fields unusable since once it captures focus, it breaks the form.

**Comment by maaslalani** [comment]
Thank you for creating this issue! It is fixed by #175.

**Comment by someblue** [comment]
I'm also intreseting on it. It help a lot to extend custom field! Is there any plan to release a tag containing this change?
